docs/mr_proc/configs.md

```diff
@@ -0,0 +1,39 @@
+## Global files
+
+---
+
+mr_proc consists of two global files for specifying local paths and recruitment manifest 
+
+---
+
+### Global configs: `global_configs.json`
+   - This is a dataset specific file and needs to be modified based on local configs and paths
+   - This file is used as an input to the `run scripts` to read, process and track available data
```

Comment:
with `run scripts` do you mean the executable scripts in the `<DATASET_ROOT>/scripts` directory?